airflow/api_connexion/openapi/v1.yaml

```diff
@@ -2910,6 +3077,7 @@ tags:
   - name: TaskInstance
   - name: Variable
   - name: XCom
+  - name: User
 
 
 externalDocs:
```

Comment:
Should we also update security section?
https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.3.md#securityRequirementObject